docs/command_line_tutorial.md

```diff
@@ -0,0 +1,90 @@
+
+- `per_instance_stats.json` contains a serialized list of `PerInstanceStats`, which contains the statistics produced for the metrics for each instance (i.e. input).
+- `run_spec.json` contains the `RunSpec`, which specifies the scenario, adapter and metrics for the run.
+- `scenario.json` contains a serialized `Scenario`, which contains the scenario for the run and specifies the instances (i.e. inputs) used.
+- `scenario_state.json` contains a serialized `ScenarioState`, which contains every request to and response from the model.
+- `stats.json` contains a serialized list of `PerInstanceStats`, which contains the statistics produced for the metrics, aggregated across all instances (i.e. inputs).
+
+The command also writes the file `benchmarking_output/runs/v1/run_specs.json`, which contains the run spec descriptions for all the runs. 
+
+`helm-run` provides additional arguments that can be used to filter out  `--models-to-run`, `--groups-to-run` and `--priority`. It can be convenient to create a large `run_specs.conf` file containing every run spec description of interest, and then use these flags to filter down the RunSpecs to actually run. As an example, the main `run_specs.conf` file used for the HELM benchmarking paper can be found [here](https://github.com/stanford-crfm/helm/blob/main/src/benchmark/presentation/run_specs.conf).
+
+## Using `helm-summarize`
+
+The `helm-summarize` reads the output files of `helm-run` and computes aggregate statistics. Run the following:
+
+```
+helm-summarize --suite v1
+```
+
+This reads the pre-existing files in `benchmark_output/runs/v1/` that were written by `helm-run` previously, and writes the following new files back to `benchmark_output/runs/v1/`:
+
+- `summary.json` contains a serialized `ExecutiveSummary` with a date and suite name.
+- `runs.json` contains serialized list of `Run`, which contains the run path, run spec and adapter spec and statistics for each run.
+- `groups.json` contains a serialized list of `Table`, each containing information about groups in a group category.
+- `groups_metadata.json` contains a list of all the groups along with a human-readable description and a taxonomy.
+
+Additionally, for each group and group-relavent metric, it will output a pair of files: `benchmark_output/runs/v1/groups/latex/<group_name>_<metric_name>.tex` and `benchmark_output/runs/v1/groups/latex/<group_name>_<metric_name>.json`. These files contain the statistics for that metric from each run within the group.
+
+**Note**: Because `helm-summarize` reads the `run_specs.json` that was produced by `helm-run` and `run_specs.json` is overwritten by every invocation of `helm-run`, `helm-summarize` will only process runs from the most recent invocation of `helm-run`.
+
+## Using `helm-server`
+
+Finally, the `helm-server` command launches a web server to visualize the output files of `helm-run` and `helm-benchmark`. Run:
```

Comment:
Say that you can serve the results statically - you can copy them to an existing directory if they wish (interestingly, we could host all the outputs on CodaLab even).

Reply:
This feels a bit like it should be its own tutorial... right now users have to copy the static assets from the repository or from the package installation directory, because there's no easy way to get the static assets out of a `pip install` package. Filed #1201 for this.